src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -335,6 +378,43 @@ var ChangeEventPlugin = {
 
   eventTypes: eventTypes,
 
+  _isInputEventSupported: isInputEventSupported,
+
+  willDeleteListener(inst) {
+    var targetNode;
+
+    // An uglier internal check to avoid a try/catch
+    // if the instance is unmounted the node will already be removed
+    if (inst._nativeNode !== null) {
```

Comment:
When unmounting the node is already gone by the time this hook is called so the `getNodeFromInstance` call throws.

This seems like an unfortunate break in encapsulation, the alternative was to use a try/catch but I'm mostly conditioned at this point to be afraid of try/catch VM deopts.